CMakeLists.txt

```diff
@@ -50,16 +50,21 @@ endif()
 #
 install(
   TARGETS ${PROJECT_NAME}_conversion
-  EXPORT ${TARGETS_EXPORT_NAME}
-  RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
+  EXPORT export_${PROJECT_NAME}
```

Comment:
Do you have any pointer to documentation for that, or explanations ?
I mean: `TARGETS_EXPORT_NAME` is defined in jrl-cmakemodules, so if this is wrong, it should be updated there

Reply:
It was a bad copy-paste from my part. It should be removed. I am not using jrl-cmakemodules but ament here.

Reply:
Actually the logic is set by ament below line 64:
`ament_export_targets(export_${PROJECT_NAME} HAS_LIBRARY_TARGET)`
Here you define the export name.